### PR TITLE
Updated setup instructions

### PR DESCRIPTION
**Pull request by hiranya911**
Minor updates based on DiRT feedback.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,19 +135,31 @@ If you wish to skip the linter, and only run the unit tests:
...
-account key file from the "Settings > Service Accounts" page of the project, and copy it to
-`test/resources/key.json`. Also obtain the API key for the same project from "Settings > General",
-and save it to `test/resources/apikey.txt`.
-
-Some Auth integration tests require that you enable the IAM API for your Firebase/GCP project,
-and grant your service account ID the "Service Account Token Creator" role. These must be done
-via the Google Cloud Console. Refer to the
-[troubleshooting instructions](https://firebase.google.com/docs/auth/admin/create-custom-tokens#troubleshooting)
-in the official documentation for more details on how to achieve this.
+The integratin tests run against an actual Firebase project. Create a new
```
Comment (bojeil-google) [inline CONTRIBUTING.md]: integration

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,19 +135,31 @@ If you wish to skip the linter, and only run the unit tests:
...
+project in the [Firebase Console](https://console.firebase.google.com), if you
+do not already have one suitable for running the tests aginst. Then obtain the
```
Comment (bojeil-google) [inline CONTRIBUTING.md]: against

File: CONTRIBUTING.md (hunk 1 above)
Comment (hiranya911) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (hiranya911) [reply CONTRIBUTING.md]: Done

**Pull request merged**